src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
Create helpers in the global scope: isMale/ isFemale
`cosnt isMale = (person) => person.sex === MALE;`

```suggestion
    ? people.filter(({sex, died}) => isMale(sex) && Math.ceil(died / 100) === century)
    : people.filter(isMale);
```